src/namespace.c

```diff
@@ -152,7 +152,7 @@ cc_oci_str_to_ns (const char *str)
  */
 gboolean cc_oci_ns_join(struct oci_cfg_namespace *ns)
 {
-	int fd = -1;
+	int fd;
```

Comment:
1. cc_oci_ns_join is a new function
2. cppcheck reports errors if fd is initialized here
3. fd is initialized below